gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -304,77 +299,123 @@ You should see a list of services. Flex Gateway is successfully running if each
 ----
 
 // end::linux-start-commands[]
-// tag::k8s-install-helm-chart[]
+// tag::k8s-install-flex-helm-chart-title[]
 
 === Install Helm Chart into the Namespace
 
-Before installing, ensure that you have:
-
-* https://helm.sh/docs/intro/install/[Helm^], a tool used to install Flex Gateway, monitoring tools, and applications. A minimum Helm version of 3.0.0 is required. 
-+
-Refer to the <<helm-chart-options,Helm Chart Configuration Options>> for information about customizing the chart.
-
-. Add the Flex Gateway Helm repository:
+// end::k8s-install-flex-helm-chart-title[]
+// tag::k8s-install-flex-helm-chart-intro-connected[]
+Use Helm to deploy Flex Gateway to a node in your cluster and connect it to Anypoint Platform. After deploying, use Runtime Manager to verify that the gateway is present and connected to Anypoint Platform.
+// end::k8s-install-flex-helm-chart-intro-connected[]
+// tag::k8s-install-flex-helm-chart-intro-local[]
+Use Helm to deploy Flex Gateway to a node in your cluster. 
+// end::k8s-install-flex-helm-chart-intro-local[]
+// PLS don't move k8s-mode-overview
+// tag::k8s-mode-overview[]
+For information about deployments when using {reg-mode} mode, see xref:flex-gateway-k8-overview.adoc[].
+// end::k8s-mode-overview[]
+// tag::k8s-flex-helm-chart[]
+A Helm chart installs Flex Gateway, monitoring tools, and applications.
+// end::k8s-flex-helm-chart[]
+// tag::k8s-flex-helm-chart-defaults[]
+Refer to <<helm-chart-options>> for configuration options. 
+// end::k8s-flex-helm-chart-defaults[]
+
+. Add a Helm repository named `flex-gateway` for your chart: 
 +
-[source,kubernetes]
+[source,helm]
 ----
 helm repo add flex-gateway https://flex-packages.anypoint.mulesoft.com/helm
 ----
++
```

Comment:
aligning with reviewed content in the getting started guide for k8 here and elsewhere in this PR